public/index.html

```diff
@@ -41,12 +41,12 @@
         const instances = M.Sidenav.init(elems);
       });
     </script>
-    <script>
+    <!-- <script>
```

Comment:
@habibaudu Since we no longer use this, let us comment it out.

Reply:
ok